src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -313,6 +316,17 @@ public function checkAnnotations(\ReflectionClass $refl, $class)
             return $deprecations;
         }
 
+        if (
+            isset($parentAndOwnInterfaces['Serializable']) &&
```

Comment:
this should be an "is_a" check, so that this is triggered for classes whose parent interfaces extends Serializable

Reply:
The problem with a `is_a` check is that it will trigger for classes extending `\ArrayIterator` for example. And we're not gonna stop extending this core class in the core I guess.

Reply:
@nicolas-grekas Thanks for your review. I have addressed all comments. This is the last one 😄